### DISCUSSION TITLE
Error trying to get group

### DISCUSSION THREAD

**Discussion by BC123456789** [bisand/node-red-contrib-panasonic-comfort-cloud#11] [General]
Using the provided example along with my account credentials, I continue to get an error when trying to get the group, see logs;

9 Feb 15:24:41 - [info] Started flows
9 Feb 15:24:48 - [info] [pcc-groups:groups] Obtained a new access token.
9 Feb 15:24:49 - [error] [pcc-groups:groups] Error: An error ocurred while trying to get group: {}

If I put incorrect credentials in, I get an different error stating they are wrong, so I believe the error is not credential related.

Any ideas?

FYI the IOS app works fine.

Cheers
Ben

**Comment by bisand** [comment]
This seems to be a bug. I will investigate and get back to you as soon as I have something to report.

Thanks!

**Comment by hannuterasloihde** [comment]
I'm having the same issue

**Comment by bisand** [comment]
Hi,

A new version (1.3.3) has just been released and should address the issues you are experiencing. Please install the new version to see if that fixes the problem.

Thanks!

**Reply by mamoebius** [reply]
Hello,
I also tested it. Node "groups" works now, as does Node "command". Thanks. Only the node "device" brings the following error:

"Error: An error ocurred while trying to get device. Check Device ID or credentials: {"message":"Have no authority to the request","code":4300,"httpCode":403,"statusCode":403,"statusMessage":""}"

Could you please take a look at what's wrong. Or is there another way to get the measured inside temperature via groups. This is displayed in the top left of the app. I need this very urgently because I want to program a power setting with it.
Mfg Möbius

**Reply by mamoebius** [reply]
Hi, I found the bug. Before the ID comes the device designation plus a +,
thanks
Mfg Möbius
